rails_docker/Dockerfile

```diff
@@ -12,58 +12,63 @@ ENV RACK_ENV=$RUBY_ENV \
     BUILD_ENV=$BUILD_ENV \
     ASSET_HOST=$ASSET_HOST
 
-ENV APP_HOME=/#{app_name} \
-    PORT=80
+ENV APP_HOME=/usr/src/#{app_name}
+ENV PORT=80
 
 ENV BUNDLE_GEMFILE=$APP_HOME/Gemfile \
     BUNDLE_JOBS=4 \
     BUNDLE_PATH="/bundle"
 
-ENV NODE_VERSION="8"
-
 ENV LANG="en_US.UTF-8" \
     LC_ALL="en_US.UTF-8" \
     LANGUAGE="en_US:en"
 
-RUN apt-get update -qq && \
-    apt-get install -y --no-install-recommends apt-transport-https curl gnupg
-
-# Add Yarn repository
-ADD https://dl.yarnpkg.com/debian/pubkey.gpg /tmp/yarn-pubkey.gpg
-RUN apt-key add /tmp/yarn-pubkey.gpg && rm /tmp/yarn-pubkey.gpg && \
-    echo "deb http://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
-
-# Add the PPA (personal package archive) maintained by NodeSource
-# This will have more up-to-date versions of Node.js than the official Debian repositories
-RUN curl -sL https://deb.nodesource.com/setup_"$NODE_VERSION".x | bash -
-
-# Install general required core packages, Node JS related packages and Chrome (testing)
-RUN apt-get update -qq && \
-    apt-get install -y --no-install-recommends build-essential libpq-dev nodejs yarn && \
-    apt-get install -y --no-install-recommends rsync locales chrpath pkg-config libfreetype6 libfontconfig1 git cmake wget unzip && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
-# Set up the Chrome PPA and install Chrome Headless
+# Install dependencies
+RUN apk add \
+      --no-cache \
+      --update build-base \
+        build-base \
+        linux-headers \
+        imagemagick \
+        ruby \
+        ruby-bundler \
+        ruby-irb \
+        libxml2-dev \
+        libxslt-dev \
+        libffi-dev \
+        zlib-dev \
+        git \
+        postgresql \
```

Comment:
postgresql is generally set up as a separate service hence does not need be set up in the web app container

Reply:
Ah yes, of course. Will remove.

Reply:
turns out we need the `postgresql-dev` package to be able to install the `pg` gem.